app/scripts/modules/core/src/styleguide/src/styles/postcss.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     'postcss-scopeit': {scopeName: 'styleguide'},
     'postcss-style-guide': {
       project: 'Spinnaker',
-      dest: path.join(__dirname, '..', '..', 'src', 'public', 'styleguide.html'),
+      dest: path.join(__dirname, '..', '..', 'public', 'styleguide.html'),
```

Comment:
Can you verify you can still run the library build with this change?
(navigate to `/app/scripts/modules/core` and run `npm run lib`)